src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -2073,7 +2075,7 @@ private DynamoLineGeometryModel3D CreateLineGeometryModel3D(HelixRenderPackage r
                 Transform = new MatrixTransform3D(rp.Transform.ToMatrix3D()),
                 Color = Colors.White,
                 Thickness = thickness,
-                IsHitTestVisible = false,
+                IsHitTestVisible = true,
```

Comment:
Enables geometry selection for line/curve geometry.

Reply:
any performance impact you see?

Reply:
Not that I have noticed.